charts/ccd-data-store-api/Chart.yaml

```diff
@@ -2,7 +2,7 @@ description: Helm chart for the HMCTS CCD Data Store
 name: ccd-data-store-api
 apiVersion: v2
 home: https://github.com/hmcts/ccd-data-store-api
-version: 1.7.2
+version: 1.7.3
```

Comment:
@henrykcgi  - The java version below went from 3.4.4 to 3.4.2 as part of RDM-9052. Was that deliberate?

Reply:
No. It should be 3.4.4, it should not be below.. and revisions should match.. also the version.1.7.2